apps/qubit/modules/default/templates/_privacyMessage.php

```diff
@@ -0,0 +1,12 @@
+<script type="text/javascript" charset="utf-8" src="/js/privacyMessage.js"></script>
```

Comment:
I think it's better to add it in the component, so it gets added to the head section instead if in the body. E.g.:

https://github.com/artefactual/atom/blob/qa%2F2.5.x/apps/qubit/modules/informationobject/actions/indexAction.class.php#L122

Reply:
I have tried to make this work by adding the javascript in the controller but it has no effect even after clearing cache, restarting services etc. I believe that this does not work for classes extending sfComponent. I have tried including as in your example which extends sfAction, but it has no effect - the js is not included in the rendered page. I can find other classes in AtoM which extend sfComponent that have also included the js in the template.

Reply:
I have also tried loading this using the view.yml config file for default module, but this has the side effect of loading the js on every AtoM page access, even when it is not needed. Placing this in the template was the only way I could find to include this js successfully, but only when the banner is actually displayed and the js is required.

Reply:
I didn't know that, thanks!